osu.Game/Overlays/ModSelection/ModSelection.cs

```diff
@@ -0,0 +1,520 @@
+                                },
+                                new FlowContainer
+                                {
+                                    // upper container
+                                    RelativeSizeAxes = Axes.Both,
+                                    Width = padded_width,
+                                    Direction = FlowDirection.VerticalOnly,
+                                    Anchor = Anchor.TopCentre,
+                                    Origin = Anchor.TopCentre,
+
+                                    Children = new Drawable[]
+                                    {
+                                        new SpriteText
+                                        {
+                                            Text = @"Gameplay Modes",
+                                            TextSize = 30,
+                                            Padding = new MarginPadding { Top = 10, Bottom = 8 },
+                                        },
+                                        new SpriteText
+                                        {
+                                            Text = @"Mods provide different ways to enjoy gameplay. Some have an effect on the score you can achieve during ranked play.",
+                                            TextSize = 20,
+                                        },
+                                        new SpriteText
+                                        {
+                                            Text = @"Others are just for fun",
+                                            TextSize = 20,
+                                        }
+                                    }
+                                }
+                            }
+                        },
+                        // main container
+                        new FlowContainer
+                        {
+                            Anchor = Anchor.TopCentre,
+                            Origin = Anchor.TopCentre,
+                            RelativeSizeAxes = Axes.Both,
+                            Width = padded_width,
+                            Height = 0.65f,
+                            Direction = FlowDirection.VerticalOnly,
+                            Children = new Drawable[]
+                            {
+                                new SpriteText
+                                {
+                                    Margin = new MarginPadding { Top = 15, Bottom = section_margin },
+                                    Text = @"Gameplay Difficulty Reduction",
+                                    TextSize = textsize_section_info,
+                                    Font = @"Exo2.0-Bold",
+                                },
+                                easymods = new FlowContainer
+                                {
+                                    Margin = new MarginPadding { Top = section_margin },

```

Comment:
take a look at `Spacing` instead, may work better than specifying this multiple times.